src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -930,6 +937,7 @@ class kernel_command : public xrt_core::command
 private:
   std::shared_ptr<device_type> m_device;
   xrt_core::hw_queue m_hwqueue;  // hwqueue for command submission
+  xrt::hw_context m_hwctx;       // hw_context for command
```

Comment:
I hope kernel_command is specific to one Kernel.

Reply:
yes